config/application.yml.example

```diff
@@ -162,6 +164,7 @@ test:
   attribute_cost: '800$8$1$' # SCrypt::Engine.calibrate(max_time: 0.01)
   attribute_encryption_key: '2086dfbd15f5b0c584f3664422a1d3409a0d2aa6084f65b6ba57d64d4257431c124158670c7655e45cabe64194f7f7b6c7970153c285bdb8287ec0c4f7553e25'
   attribute_encryption_key_queue: '["old-key-one", "old-key-two"]'
+  available_locale: 'en es'
```

Comment:
Should be plural per:
`config.i18n.available_locales = Figaro.env.available_locales.split(' ')`